specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/Assessments/GetAssessment_example.json

```diff
@@ -0,0 +1,32 @@
+{
+    "parameters": {
+        "api-version": "2019-01-01-preview",
+        "resourceId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss1",
+        "assessmentName": "21300918-b2e3-0346-785f-c77ff57d243b"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss2/providers/Microsoft.Security/assessments/21300918-b2e3-0346-785f-c77ff57d243b",
+                "name": "21300918-b2e3-0346-785f-c77ff57d243b",
+                "type": "Microsoft.Security/assessments",
+                "properties": {
+                    "resourceDetails": {
+                      "assessedResourceSource": "Azure",
+                      "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss2/providers/Microsoft.Security/assessments/21300918-b2e3-0346-785f-c77ff57d243b"
+                    },
+                    "displayName": "Install endpoint protection solution on virtual machine scale sets",
+                    "status": "NotApplicable",
+                    "statusReasonCode": "OffByPolicy",
+                    "statusReasonDescription": "The effective policy for the assessment was evaluated to off - use Microsoft.Authorization/policyAssignments to turn this assessment on",

```

Comment:
All of these properties are prefixed with `status`. Should we combine them into a single object to make it more readable?

Reply:
done